app/src/main/java/me/morglenrazor/imdbcollector/ui/base/BaseFragment.kt

```diff
@@ -0,0 +1,15 @@
+package me.morglenrazor.imdbcollector.ui.base
+
+import android.widget.Toast
+import kotlinx.android.synthetic.main.activity_main.*
+import me.morglenrazor.imdbcollector.presentaiton.base.BaseView
+import moxy.MvpAppCompatFragment
+
+abstract class BaseFragment : MvpAppCompatFragment(), BaseView{
+
+    override fun showMessage(msg: String) {
+            val toast = Toast.makeText(activity, msg, Toast.LENGTH_LONG)
```

Comment:
U can remove showMessage from your Activity. If you have that method at Fragment. U should use only one method.